rabbitmq-setup.sh

```diff
@@ -0,0 +1,17 @@
+echo 'deb http://www.rabbitmq.com/debian/ testing main' | sudo tee /etc/apt/sources.list.d/rabbitmq.list && sudo apt-get update && sudo apt-get install rabbitmq-server
+sudo apt-get install cmake libboost-dev openssl libssl-dev libblkid-dev e2fslibs-dev libboost-all-dev libaudit-dev
```

Comment:
Let's also install the following things too in case the user is starting with a fresh VM.
- `software-properties-common`

- `build-essential`

 

- `mesa-common-dev`

 

- `libgl1-mesa-dev`